packages/tbds-forms/lib/form.scss

```diff
@@ -0,0 +1,78 @@
+.tbds-form__row + .tbds-form__row {
+  margin-top: 2rem;
+}
+
+.tbds-form__label {
+  display: block;
+  font-weight: $tbds-font-weight-bold;
+  margin-bottom: 0.5rem;
+}
+
+.tbds-form__help-text {
+  color: #666;
+  font-weight: $tbds-font-weight-normal;
+}
+
+.tbds-form__help-text--block {
+  display: block;
+  margin-top: 0.3rem;
+}
+
+.tbds-form__text-input {
+  appearance: none;
+  background-color: #fff;
+  border: 1px solid #888;
+  border-radius: 4px;
+  font-size: 16px;
+  margin: 0;
+  padding: 0.5rem 0.8rem;
+  width: 100%;
+
+  &:focus {
+    border-color: $tbds-blue;
+    box-shadow: 0 0 0 1px $tbds-blue;
+    outline: none;
+  }
+}
+
+.tbds-form__textarea {
+  appearance: none;
+  background-color: #fff;
+  border: 1px solid #888;
+  border-radius: 4px;
+  font-size: 16px;
+  line-height: 1.4;
+  margin: 0;
+  min-height: 2.25rem;
+  padding: 0.5rem 0.8rem;
+  resize: vertical;
+  width: 100%;
+
+  &:focus {
+    border-color: $tbds-blue;
```

Comment:
Did we have a semantic-use variable name for something like this, instead of the hue-named variable?

Reply:
Not yet! But definitely should have one in the future!